lib/etcdec.cxx

```diff
@@ -1,9 +1,9 @@
 /*
```

Comment:
We should move this file to `external/etcdec` or similar so it is not reformatted. Strict adherence to the terms of the license required not modifying the file.

Reply:
Thanks for the info, I didn't notice this is also an external file.